docs/LV_client/tutorials/wheel_integration.md

```diff
@@ -0,0 +1,31 @@
+# Wheel integration
+
```

Comment:
Can we add something here about Wheel Integration, something like: Labview's platform enables users to plug-in(there's probably a better word for that) a wheel (specific wheel or any wheel?) in the simulation. In the vi package manager, there two examples are available for use with a wheel: mono_closed_loop_example_wheel.vi and mono_replay_example_wheel.vi. (add more details here about hardware options if you'd like)